submitted_models/jhuapl_edge540xl_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,45 @@
+# Edge540XL
+This specifications.md file is a description for the JHU/APL Edge540XL with Sensor Configuration 1. This robot may be launched using a `ign launch` command with the variable name `jhuapl_edge540xl_sensor_config_1`.
+
+## Description
+Edge540XL is an acrobatic fixed-wing UAV.
+In this configuration Edge540XL is equipped with the following sensors: forward facing color & depth camera.
+
+
+## Usage Instructions
+The vehicle can be launched with the following command:
+```
+ign launch -v 4 competition.ign robotName1:=edge540xl robotConfig1:=JHUAPL_EDGE540XL_SENSOR_CONFIG_1  circuit:=cave worldName:=simple_cave_01 localModel:=true
+```
+
+At this point you can control the vehicle model with a actuator command, for example:
+```
+rostopic pub -r10 /edge540xl/input mav_msgs/Actuators "angular_velocities: [0.3, 0.3, 0.3, 15.0]"
+```
+This commands the angular velocities of the aileron, elevator, and rudder, with thrust at the end.
```

Comment:
I launched using the command above and was unable to see any response from the vehicle when commanding the actuators.